crates/sui-config/src/node.rs

```diff
@@ -268,7 +268,7 @@ pub struct ExecutionTimeObserverConfig {
 impl ExecutionTimeObserverConfig {
     pub fn observation_channel_capacity(&self) -> NonZeroUsize {
         self.observation_channel_capacity
-            .unwrap_or(nonzero!(128usize))
+            .unwrap_or(nonzero!(1_024usize))
```

Comment:
i find `1_024` to be the weirdest looking thing i've ever seen, but that can be a me problem 😂